web/src/beta/features/ProjectSettings/innerPages/PublicSettings/AliasSettings/EditPanel.tsx

```diff
@@ -22,30 +22,29 @@ const EditPanel: FC<AliasSettingProps> = ({
 }) => {
   const t = useT();
   const theme = useTheme();
+  const { useProjectAliasCheckQuery } = useProjectFetcher();
+
   const [localAlias, setLocalAlias] = useState("");
   const [warning, setWaring] = useState(false);
-  const [, setNotification] = useNotification();
+
+  const { checkProjectAlias } = useProjectAliasCheckQuery(localAlias);
 
   const handleChange = useCallback(
     (value: string) => {
       if (value === alias) {
```

Comment:
Please await alias check here.

Reply:
You didn't wait for the result?
Personally i'll suggest to use checkAlias directly here, don't use prop `aliasValid` & `onCheckAlias`